client/src/pages/teds/charts/utils/UseBuildQuery.js

```diff
@@ -0,0 +1,21 @@
+function buildQuery(body, s) {
```

Comment:
I would have done a unique file called `hooks.js` with the 3 fonctions. Usually, the hook name starts with a lowercase and withe `use` like `useAnything`.